src/monitoring/tests/path_patterns.rs

```diff
@@ -0,0 +1,25 @@
+use crate::monitoring::performance::extract_chain_id;
+
+#[test]
+fn chain_dependent_endpoint() {
```

Comment:
Can we add a test with a `uri` that has no `v1` and another that has not `chains`?